boards/arc/nsim/nsim_hs_mpuv6.yaml

```diff
@@ -1,6 +1,6 @@
 identifier: nsim_hs_mpuv6
 name: HS (with MPU v6) nSIM simulator
-type: mcu
+type: sim
```

Comment:
I'm wondering if it's used somewhere in addition to test filtering?

All ARC `nsim_*` platforms support execution on both simulator (nSIM) and on real HW (FPGA based) which is done with multiple runners support.  We don't want to break this :)

Reply:
> I'm wondering if it's used somewhere in addition to test filtering?

It is. This way we can configure tests to only run on simulators/qemu or we can limit some tests to real hardware, now this might be an issue of the simulator can also run on FPGAs for example, we have this case already with some ARM platforms, I guess we need to support this case as well, but AFAIL nsim_* is primeraly used as a simulator (it is in the name ;-)